<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="EXE_Icons.qrc">
    <normaloff>:/exe_icon/EXE Icons/FRD_icon.png</normaloff>:/exe_icon/EXE Icons/FRD_icon.png</iconset>
  </property>
  <widget class="QPushButton" name="Login_login">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Log in</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Login_cancel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Passcode</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Username_line">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Enter username here.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Passcode_line">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Enter passcode here.</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_savepasscode">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>131</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Passcode</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Sitka Banner</family>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Fractal Designer</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="EXE_Icons.qrc"/>
 </resources>
 <connections/>
</ui>
